tools/test-framework/src/chain/driver.rs

```diff
@@ -453,23 +478,33 @@ impl ChainDriver {
         user: &Wallet,
         target_amount: u64,
         denom: &Denom,
+    ) -> Result<(), Error> {
+        self.assert_eventual_wallet_addr_amount(&user.address, target_amount, denom)
+    }
+
+    /**
+       Assert that a wallet should eventually have the expected amount in the
+       given denomination.
+    */
+    pub fn assert_eventual_wallet_addr_amount(
```

Comment:
It looks like we can just change `assert_eventual_wallet_amount` to accept `WalletAddress` instead of `Wallet`.

Reply:
Right, I actually meant to do that and forgot to do it after I had this initial version working.

Reply:
[1b90428](https://github.com/informalsystems/ibc-rs/pull/1931/commits/1b90428abd00ec644f2caee1d489212489804273)